### PR TITLE
 Adding zh-CN, indexing zh-CN and pl 

### PR DESCRIPTION
**Pull request by vesper-mozilla**
As per request from wxie, our zh-CN lead, I'm adding initial zh-CN content for SUMO.

Also adding the missing indexing for Polish as a small clean up add-on.

### CODE REVIEW

**Comment by MikkCZ** [comment]
To make the content appear in the Gitbook, you need to list it in https://github.com/mozilla-l10n/styleguides/blob/master/SUMMARY.md too.

Actually I wonder if the list of style guides in the main README.md makes much sense. Viewing it on GitHub, I can open the desired folder in the list. When I need a better readable view with the locale names etc., I use the Gitbook anyway.

**Comment by flodolo** [comment]
Please also fix [Travis (syntax) failures](https://travis-ci.org/mozilla-l10n/styleguides/builds/344847788?utm_source=github_status&utm_medium=notification) ;-)

File: zh-CN/sumo.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
-
 # Style Guide for Simplified Chinese（zh-cn）- 中文（简体）本地化翻译须知
```
Comment (flodolo) [inline zh-CN/sumo.md]: Nit: zh-CN should be consistently spelled as such, not zh-cn.

**Comment by vesper-mozilla** [comment]
Is this OK to merge?

**Comment by flodolo** [comment]
> Is this OK to merge?

As long as tests pass, feel free to merge ;-)

**Pull request merged**